### ISSUE TITLE
Some questions about the processing of BVH data.

### ISSUE THREAD

**Issue by zhou-fy** [Wasserwecken/bvhio#13]
Hello, I am not familiar with the processing of BVH data and would like to consult with you. Do you know how to unify different BVH files into the same hierarchy? Can it be achieved with the bvhio you proposed?
I encountered a problem when re-implement the code of a paper. The BVH files in my dataset have 31 joints, but the BVH files in the paper's dataset only have 24 joints. Therefore, I would like to ask is there any way to convert a bvh file with 31 joints into a bvh file with 24 joints?
The specific files are like follows.
[bvhData.zip](https://github.com/Wasserwecken/bvhio/files/15023515/bvhData.zip)
Thank you very much for your answer!

**Comment by Wasserwecken** [comment]
> Hello, I am not familiar with the processing of BVH data and would like to consult with you. Do you know how to unify different BVH files into the same hierarchy? Can it be achieved with the bvhio you proposed?

Yes and no, I've added an example to the readme how to chain animations into one file. BUt you cannot merge different rest poses / hierarchies. Chaining animations from one file into one only works if the rest pose / hierarchy is EXACTLY the same. https://github.com/Wasserwecken/bvhio?tab=readme-ov-file#merge-bvh-files

> > The BVH files in my dataset have 31 joints, but the BVH files in the paper's dataset only have 24 joints. Therefore, I would like to ask is there any way to convert a bvh file with 31 joints into a bvh file with 24 joints?

No, this library is not magic. It cannot automatically convert your hierarchy. BUT this library can be used to detach and rearange individual joints with their animations. But I cannot help you with that. You have to play around with the methods `.attach()` and `.detach()`. For transforming the rest pose and animation have a look to here: https://gist.github.com/Wasserwecken/0802f3a678931e8408f7b78ecb99b00a
